docs/advanced/testing.md

```diff
@@ -169,6 +181,43 @@ const mockClient = () => {
 };
 ```
 
+## Subscriptions
+
+Testing subscription can be done by simulating the arriving of new data after an amount of time. To do that you can use `interval` from `wonka` and for each value we can set the response that we want to mock.
+
+If you prefer to have more control on when the new data is arriving you can use the `makeSubject` utility from `wonka`. You can see more details in the next section.
+
+Here's an example of testing a list component which use subscription.
+
+```tsx
+const mockClient = {
+  executeSubscription: jest.fn(() =>
+    pipe(
+      interval(200),
+      map((i: number) => ({
+        data: { posts: { id: i, title: 'Post title', content: 'This is a post' } },
+      }))
+    )
+  ),
```

Comment:
```suggestion
  executeSubscription: jest.fn(query =>
    pipe(
      interval(200),
      map((i: number) => ({
        // To mock a full result, we need to pass a mock operation back as well
        operation: {
          operationName: 'subscription,
          context: {},
          ...query,
        },
        data: { posts: { id: i, title: 'Post title', content: 'This is a post' } },
      }))
    )
  ),
```

The `operation`

Reply:
Just as a note, I'll need to come back to this and add more notes on how to create `Operation`s for mocking purposes, since that's pretty important. This is a quick and dirty solution that'll work for most users though